api/v1alpha1/hypershiftdeployment_types.go

```diff
@@ -95,6 +95,7 @@ type HypershiftDeploymentSpec struct {
 	TargetNamespace string `json:"targetNamespace"`
 
 	//TargetManagedCluster specify which managedcluster's namespace the manifestwork would be deployed.
+	//Only applies when Spec.Override: MANIFESTWORK
 	//If not specified, it uses the current namespace.
```

Comment:
I think we should delete this line, and note only ManifestWork case needs this field.

Reply:
btw, I'm wondering if `targetManagedCluster` is a user-friendly name, I mean we actually use the `management` word in a lot of places, right?

Reply:
We're actually changing ManagementCluster to Hosting Service Cluster from a UX and discussion point of view.

Reply:
Adjusted the txt.